bigquery/docs/index.rst

```diff
@@ -1,408 +1,43 @@
-Cloud KMS for the destination table.
-
-.. literalinclude:: snippets.py
-   :language: python
-   :dedent: 4
-   :start-after: [START bigquery_query_destination_table_cmek]
-   :end-before: [END bigquery_query_destination_table_cmek]
-
-Queries
-~~~~~~~
-
-
-Querying data
-^^^^^^^^^^^^^
-
-Run a query and wait for it to finish:
-
-.. literalinclude:: snippets.py
-   :language: python
-   :dedent: 4
-   :start-after: [START bigquery_query]
-   :end-before: [END bigquery_query]
-
-
-Run a dry run query
-^^^^^^^^^^^^^^^^^^^
-
-.. literalinclude:: snippets.py
-   :language: python
-   :dedent: 4
-   :start-after: [START bigquery_query_dry_run]
-   :end-before: [END bigquery_query_dry_run]
-
-
-Writing query results to a destination table
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-See BigQuery documentation for more information on
-`writing query results <https://cloud.google.com/bigquery/docs/writing-results>`_.
-
-.. literalinclude:: snippets.py
-   :language: python
-   :dedent: 4
-   :start-after: [START bigquery_query_destination_table]
-   :end-before: [END bigquery_query_destination_table]
-
-
-Run a query using a named query parameter
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-See BigQuery documentation for more information on
-`parameterized queries <https://cloud.google.com/bigquery/docs/parameterized-queries>`_.
-
-.. literalinclude:: snippets.py
-   :language: python
-   :dedent: 4
-   :start-after: [START bigquery_query_params_named]
-   :end-before: [END bigquery_query_params_named]
-
-
-Jobs
-~~~~
-
-List jobs for a project
-^^^^^^^^^^^^^^^^^^^^^^^
-
-Jobs describe actions performed on data in BigQuery tables:
-
-- Load data into a table
-- Run a query against data in one or more tables
-- Extract data from a table
-- Copy a table
-
-.. literalinclude:: snippets.py
-   :language: python
-   :dedent: 4
-   :start-after: [START bigquery_list_jobs]
-   :end-before: [END bigquery_list_jobs]
-
-
-Using BigQuery with Pandas
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Retrieve BigQuery data as a Pandas DataFrame
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-As of version 0.29.0, you can use the
-:func:`~google.cloud.bigquery.table.RowIterator.to_dataframe` function to
-retrieve query results or table rows as a :class:`pandas.DataFrame`.
-
-First, ensure that the :mod:`pandas` library is installed by running:
-
-.. code-block:: bash
-
-   pip install --upgrade pandas
-
-Alternatively, you can install the BigQuery python client library with
-:mod:`pandas` by running:
-
-.. code-block:: bash
+More Examples
+-------------
 
-   pip install --upgrade google-cloud-bigquery[pandas]
+These guides cover how to use the ``google-cloud-bigquery`` library to use
+the Google BigQuery API. See the `official Google BigQuery how-to guides
+<https://cloud.google.com/bigquery/docs/how-to>`_ for more in-depth
+information and code samples.
 
-To retrieve query results as a :class:`pandas.DataFrame`:
+BigQuery Basics
+~~~~~~~~~~~~~~~
```

Comment:
The rest of the headers should probably be H2 (`-`) instead of H3 (`~`), otherwise they'll fall under the "Example Usage" header, which was the last H2. Or if you were intending for these to be part of "Example Usage", I'd move the "More Examples" header to the bottom of the "Example Usage" section.

Reply:
Done in https://github.com/googleapis/google-cloud-python/pull/6238/commits/af8a6622aa38f5926b8470bd2f982c861c0add8f

Moved it to the bottom.